tests/guestlog/guestlog.go

```diff
@@ -105,7 +105,7 @@ var _ = Describe("[sig-compute]Guest console log", decorators.SigCompute, func()
 						foundContainer = true
 					}
 				}
-				Expect(foundContainer).To(Equal(true))
```

Comment:
Sure.

Do you know if doable to have ginkolinter integrated with bazel so we would get compile time errors over this?

/lgtm

Reply:
sure, it's tracked here: https://github.com/kubevirt/kubevirt/issues/11006